src/components/30-organisms/modal/index.js

```diff
@@ -0,0 +1,94 @@
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import closeIcon from '@axa-ch/materials/internal-icons/close.svg';
+
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAModal extends LitElement {
+  static get tagName() {
+    return 'axa-modal';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      open: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  render() {
+    const classes = {
+      'o-modal--open': this.open,
+    };
+    return html`
+      <article class="o-modal ${classMap(classes)}">
+        <div class="o-modal-window">
+          <div class="o-modal-window__close-button" @click="${this.closeModal}">
+            ${svg([closeIcon])}
+          </div>
+          <slot></slot>
+        </div>
+        <div class="o-modal-window__close-container">
+          <axa-button
+            class="o-modal-window__close-container-button"
+            @click="${this.closeModal}"
+            >Schliessen</axa-button
```

Comment:
The string "Schliessen" should be a prop because it should be translated by the app who is using the modal.